core/store/migrate/migrations/0104_terra_cascade_delete.sql

```diff
@@ -0,0 +1,25 @@
+-- +goose Up
+ALTER TABLE terra_nodes
+DROP CONSTRAINT terra_nodes_terra_chain_id_fkey;
+
+ALTER TABLE terra_nodes
+    ADD FOREIGN KEY (terra_chain_id) REFERENCES terra_chains(id) ON DELETE CASCADE;
```

Comment:
A common gotcha here is that fk targets can be unindexed. Might be worth a double check on terra_nodes (terra_chain_id) et al

Reply:
Confirmed